transformer_engine/jax/praxis/module.py

```diff
@@ -19,7 +19,6 @@
 from ..flax.module import LayerNormMLP as flax_LayerNormMLP
 from ..flax.module import Softmax
 from ..softmax import SoftmaxType
-from ..sharding import MajorShardingType, ShardingType
```

Comment:
This doesn't seem related to what the description of this PR does.
Can you add to the PR description to explain this part of the change?

Reply:
Done, add to the `Changes`.